kili/authentication.py

```diff
@@ -34,7 +34,7 @@ class KiliAuth:
     """
 
     def __init__(self,
-                 api_key=os.getenv('KILI_USER_API_KEY'),
+                 api_key=os.getenv('KILI_API_KEY'),
```

Comment:
you can remove the default values for `api_key` and `api_endpoint`, as they are already passed by the client